syne_tune/tuner.py

```diff
@@ -403,6 +403,14 @@ def save(self, folder: Optional[str] = None):
             tuner_serialized_path = Path(folder) / "tuner.dill"
         with open(tuner_serialized_path, "wb") as f:
             logger.debug(f"saving tuner in {tuner_serialized_path}")
+
+            # We need to delete the TensorboardCallback before we serialize the Tuner, to avoid runtime errors because
+            # of the MultiProcessing Queues of TensorboardX
+            for callback in self.callbacks:
+                from syne_tune.tuner_callback import TensorboardCallback
+                if isinstance(callback, TensorboardCallback):
+                    self.callbacks.remove(callback)
+
```

Comment:
Can we remove this part? We should really avoid to have this depend on special use-cases (as much as we can of-course). 

In particular here, you can avoid the serialization issue by just having `__getstate__` and `__setstate__` in your Tensorboard callback that dont save stuff you dont want that are unneeded and not serializable.

Reply:
ah perfect, that's exactly what I was looking for :-)